torch/_torch_docs.py

```diff
@@ -7999,18 +7999,47 @@ def merge_dicts(*dicts):
            r"""
 svd(input, some=True, compute_uv=True, *, out=None) -> (Tensor, Tensor, Tensor)
 
-This function returns a namedtuple ``(U, S, V)`` which is the singular value
-decomposition of a input real matrix or batches of real matrices :attr:`input` such that
-:math:`input = U \times diag(S) \times V^T`.
+Compute the singular value decomposition of either a matrix or batch of
+matrices :attr:`input`." The singular value decomposition is represented as a
+namedtuple ``(U, S, V)``, such that :math:`input = U \times diag(S) \times
+V^T`, where :math:`V^T` is the transpose of ``V``. If the inputs are batches, then
+returns batched outputs for all of ``U``, `S`` and ``V``.
 
-If :attr:`some` is ``True`` (default), the method returns the reduced
-singular value decomposition i.e., if the last two dimensions of
-:attr:`input` are ``m`` and ``n``, then the returned `U` matrix will
-contain only :math:`min(n, m)` orthonormal columns and the size of `V`
-will be :math:`(*, n, n)`.
+The original tensor can be reconstructed by::
 
-If :attr:`compute_uv` is ``False``, the returned `U` and `V` matrices will be zero matrices
-of shape :math:`(m \times m)` and :math:`(n \times n)` respectively. :attr:`some` will be ignored here.
+    U @ diag(S) @ V.T
+
+If :attr:`some` is ``True`` (default), the method returns the reduced singular
+value decomposition i.e., if the last two dimensions of :attr:`input` are
+``m`` and ``n``, then the returned `U` and `V` matrices will contain only
+:math:`min(n, m)` orthonormal columns.
+
+If :attr:`compute_uv` is ``False``, the returned `U` and `V` will be
+zero-filled matrices of shape :math:`(m \times m)` and :math:`(n \times n)`
+respectively, and the same device as :attr:`input`. The :attr:`some`
+argument has no effect when :attr:`compute_uv` is False.
+
+The dtypes of ``U`` and ``V`` are the same as :attr:`input`'s. ``S`` will
+always be real-valued, even if :attr:`input` is complex.
+
+.. warning:: ``torch.svd`` is deprecated. Please use :meth:`~torch.linalg.svd`
+             instead, which provides a better compatibility with
+             ``numpy.linalg.svd``.
+
+.. note:: **Differences with** :meth:`~torch.linalg.svd`:
+
+             * :attr:`some` is the opposite of :meth:`~torch.linalg.svd`'s
+               :attr:`full_matricies`. Note that default value for both is
+               ``True``, so the default behavior is effectively the opposite.
+
+             * it returns ``V``, whereas :meth:`~torch.linalg.svd` returns
+               ``Vh``. The result is that when using :meth:`~torch.svd` you
```

Comment:
This reference to the method is unnecessary because we're in the documentation for the torch.svd function. Its name should be used without a link.